my-app-router/src/App.jsx

```diff
@@ -0,0 +1,24 @@
+import {Routes, Route} from 'react-router-dom';
+import { Header } from "./components/Header";
+import { Footer } from "./components/Footer";
+import { Home } from './Pages/Home';
+import { Posts } from './Pages/Posts';
+import { Todos } from './Pages/Todos';
+import { Users } from './Pages/Users';
+
```

Comment:
Make `index.js` in `Pages` folder and make export all pages, ex.
export {Home} from './Home'
export {Posts} from './Posts'
export {Todos} from './Todos' 

and import all pages this on one line